Sources/PKYouboraPlayerAdapter.swift

```diff
@@ -246,6 +246,7 @@ extension PKYouboraPlayerAdapter {
                 }
             case is PlayerEvent.SourceSelected:
                 self.lastReportedResource = event.mediaSource?.playbackUrl?.absoluteString
+                self.fireStart()
```

Comment:
With this addition to call fireStart, now we will have it in 3 places: SourceSelected, Play, Replay.
But even though this is not the solution because if the source was selected but play wasn't called, the join time will be wrong and very big.

Reply:
The start is used to fire when all the resources are ready to be sent to youbora. And this is the point where you have all the resources available. If we remove the start from here it'll be sent when Play is pressed. Since the join is sent when start playing this will trigger both methods almost at the same time. But I can remove the start from here and let the start and join method to be sent almost at the same time